README.md

```diff
@@ -32,14 +32,9 @@ Tryton.
 Installation
 ------------
 
-.. code:: sh
-
     $ pip install trytond_nereid_project
 
-To install from source
-``````````````````````
-
-.. code:: sh
+## To install from source
```

Comment:
Try making this a little more smaller (H3 or triple hash) and compare it with the rst version.